### DISCUSSION TITLE
[Feature Request]: 希望添加自定义订阅，目前插件仅支持一条

### DISCUSSION THREAD

**Discussion by xiaobaiwaf** [jxxghp/MoviePilot#3399] [讨论]
### 当前程序版本

v1.5.7-2

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

希望添加自定义订阅，目前插件仅支持一条，和NASTOOL之前的一样，感觉很实用的功能，感谢作者大大

**Comment by miemieYaho** [comment]
+1
